screwdriver/screwdriver.go

```diff
@@ -110,8 +110,9 @@ type Job struct {
 
 // CommandDef is the definition of a single executable command.
 type CommandDef struct {
-	Name string `json:"name"`
-	Cmd  string `json:"command"`
+	Name      string `json:"name"`
+	Cmd       string `json:"command"`
+	AlwaysRun string `json:"alwaysRun"`
```

Comment:
Does this part convert everything to a string before going into the launcher? https://github.com/screwdriver-cd/config-parser/blob/master/lib/phase/permutation.js#L38